test/acceptance/expected-marriage-record.js

```diff
@@ -34,11 +34,14 @@ module.exports = {
     signature: 'T. Bride'
   },
   fatherOfGroom: {
-    forenames: 'Test',
+    forenames: 'Test Fog',
     surname: 'FATHER',
     occupation: 'Father'
   },
   fatherOfBride: {
+    forenames: 'Test Fob',
+    surname: 'BRIDE',
+    occupation: 'Father'
```

Comment:
We need to get these changes into the wrangler so that the tests will still work end-to-end.

Reply:
Done.